onnx/helper.py

```diff
@@ -43,6 +43,7 @@
     defs,
     mapping,
 )
+from onnx.subbyte_helper import float32x2_to_4bitx2
```

Comment:
I would refrain from creating any new modules with "helper" in their names because it blurs its responsibilities.